source/localizable/_code-of-conduct.en.md

```diff
@@ -1,39 +1,39 @@
 Purpose
 -------
 
-A primary goal of all the conferences and user groups that refer to this Code of Conduct is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status and religion (or lack thereof).
+A primary goal of all the conferences and user groups that follow this Code of Conduct is to create an inclusive community that welcomes and supports as many people with various backgrounds and identities as possible. As such, we are committed to build a friendly and safer environment for all. These various backgrounds and identities include (but are not limited to) gender identity and expression, sex, sexual orientation, educational level, mental and physical ability, immigration status, race, ethnicity, national origin, social and economic status, age, size and religion (or lack thereof).
 
 This Code of Conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
 
-We invite all those who participate in our events to help us create safe and positive experiences for everyone.
+We invite all those who participate in our events to help us create safer and positive experiences for everyone.
```

Comment:
There's been a lot of discussion about the idea of safe spaces. Basically, we will not be able to create really safe spaces. For example, even within women-only spaces, racist/transphobic/ableist/classist etc incidents can still happen. Intersectionality is often overlooked. 
A *safe* space gives the impression that we could be free of microagressions and power structures within that space - while a *safer* space is one where we are aware of privileges and marginalization and try to deal with them.